app/jobs/create_or_update_consistency_check_job.rb

```diff
@@ -5,7 +5,7 @@ class CreateOrUpdateConsistencyCheckJob < ApplicationJob
 
   def perform(trainee)
     contact = Dttp::Contacts::Fetch.call(trainee: trainee)
```

Comment:
Could we change `Dttp::Contacts::Fetch.call` to have the same interface as `Dttp::PlacementAssignments::Fetch.call`, as in Dttp::Contacts::Fetch.call(contact_dttp_id: traine.dttp_id)?

Wondering if both classes should just have the keyword argument `dttp_id` since the class name already defines the context.

Reply:
Added a change to use `dttp_id` as a keyword arg across all ::Fetch service calls.